src/Services/UserService.ts

```diff
@@ -32,11 +45,24 @@ export class UserService {
         if (user) {
             const authenticatedUser = bcrypt.compareSync(password, user['hash_password']);
             if (authenticatedUser) {
-                const token = jwt.sign({id: user['id']}, config.jwtKey, {expiresIn: config.jwtExpiry});
+                const token = jwt.sign({id: user['id']}, config.jwtKey, this.createJwtOptions(source));
```

Comment:
`user.id` better
the above example with hash_password is because of the special char...